src/sentry/notifications/platform/target.py

```diff
@@ -92,3 +124,41 @@ def integration_installation(self) -> IntegrationInstallationT:
             model=self.integration,
             organization_id=self.organization_integration.organization_id,
         )
+
+
+@dataclass
+class SerializedNotificationTarget:
```

Comment:
Nit (and also my fault) NotificationTargetDto is probably a more appropriate name for this 😅 The dict form is really the serialized form of the target.